src/OrchardCore.Modules/OrchardCore.ContentFields/Migrations.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using OrchardCore.Data.Migration;
+using OrchardCore.ContentFields.Indexing;
+
+namespace OrchardCore.ContentFields
+{
+    public class Migrations : DataMigration
+    {
+        public int Create()
+        {
+            SchemaBuilder.CreateMapIndexTable(nameof(TextFieldIndex), table => table
+                .Column<string>("ContentType", column => column.WithLength(255))
+                .Column<string>("ContentPart", column => column.WithLength(255))
+                .Column<string>("ContentField", column => column.WithLength(255))
+                .Column<bool>("Published")
+                .Column<bool>("Latest")
+                .Column<string>("Text", column => column.Nullable().WithLength(4000))
+                .Column<string>("RichText", column => column.Nullable().Unlimited())
+            );
+
+            SchemaBuilder.CreateMapIndexTable(nameof(BooleanFieldIndex), table => table
+                .Column<string>("ContentType", column => column.WithLength(255))
+                .Column<string>("ContentPart", column => column.WithLength(255))
+                .Column<string>("ContentField", column => column.WithLength(255))
+                .Column<bool>("Published")
+                .Column<bool>("Latest")
+                .Column<bool>("Boolean", column => column.Nullable())
+            );
+
+            SchemaBuilder.CreateMapIndexTable(nameof(NumericFieldIndex), table => table
+                .Column<string>("ContentType", column => column.WithLength(255))
+                .Column<string>("ContentPart", column => column.WithLength(255))
+                .Column<string>("ContentField", column => column.WithLength(255))
+                .Column<bool>("Published")
+                .Column<bool>("Latest")
+                .Column<decimal>("Numeric", column => column.Nullable())
+            );
+
+            SchemaBuilder.CreateMapIndexTable(nameof(DateTimeFieldIndex), table => table
+                .Column<string>("ContentType", column => column.WithLength(255))
+                .Column<string>("ContentPart", column => column.WithLength(255))
+                .Column<string>("ContentField", column => column.WithLength(255))
+                .Column<bool>("Published")
+                .Column<bool>("Latest")
+                .Column<DateTime>("DateTime", column => column.Nullable())
+            );
+
+            SchemaBuilder.CreateMapIndexTable(nameof(DateFieldIndex), table => table
+                .Column<string>("ContentType", column => column.WithLength(255))
+                .Column<string>("ContentPart", column => column.WithLength(255))
+                .Column<string>("ContentField", column => column.WithLength(255))
+                .Column<bool>("Published")
+                .Column<bool>("Latest")
+                .Column<DateTime>("Date", column => column.Nullable())
+            );
+
+            SchemaBuilder.CreateMapIndexTable(nameof(ContentPickerFieldIndex), table => table
```

Comment:
Please add database indices for each of the columns

Reply:
done